library/src/main/java/com/mapzen/valhalla/Router.kt

```diff
@@ -3,14 +3,14 @@ package com.mapzen.valhalla
 interface Router {
 
     enum class Language(private val languageTag: String) {
-        CS_CZ("cs-CZ"),
-        DE_DE("de-DE"),
-        EN_US("en-US"),
-        PIRATE("en-US-x-pirate"),
-        ES_ES("es-ES"),
-        FR_FR("fr-FR"),
-        IT_IT("it-IT"),
-        HI_IN("hi-IN");
+        CS_CZ("cs"),
```

Comment:
In the future we will have narrative languages that will need the location to know which one to use. I was wondering if you want to keep specific lang-location and allow lang only (or as a fall back on client side) - I don't think we want the server to make this assumption for clients. 
Also, do you want to add in ca-ES and sl-SI - they are relatively new